src/metabase/public_settings.clj

```diff
@@ -683,3 +683,9 @@
   (deferred-tru "Prefix for upload table names")
   :visibility   :authenticated
   :type         :string)
+
+(defsetting bcc-enabled?
+  (deferred-tru "Whether or not bcc emails are enabled, default behavior is that it is")
+  :visibility :authenticated
```

Comment:
Can visibility just be `:settings-manager`? I don't think all users need to be able to read this setting since it's only fetched on the backend, right?

[Here's](https://github.com/metabase/metabase/blob/master/src/metabase/models/setting.clj#L1005-L1011) a handy chart for what the different visibility levels mean for settings.